### ISSUE TITLE
Page Stylesheet

### ISSUE THREAD

**Issue by Siecje** [Siecje/agora-api#1]
A page will have a stylesheet which will control how the comments are displayed.

**Comment by Siecje** [comment]
[branch](https://github.com/Siecje/agora-api/tree/1_page_stylesheets)

I tried 

``` Python
class RelatedUserSerializer(serializers.HyperlinkedModelSerializer):
    class Meta:
        model = get_user_model()
        fields = ('id', 'username', 'url')


class PageSerializer(serializers.ModelSerializer):
    user = RelatedUserSerializer(read_only=True)

    class Meta:
        model = Page
        fields = ('id', 'name', 'stylesheet', 'user', 'created', 'updated')

class PageViewSet(viewsets.ModelViewSet):
    lookup_field = 'id'
    parser_classes = (MultiPartParser, FormParser,)
    permission_classes = (permissions.IsAuthenticatedOrReadOnly,)
    queryset = Page.objects.all()
    serializer_class = PageSerializer

    def perform_create(self, serializer):
        if self.request.data.get('stylesheet'):
            serializer.save(stylesheet=self.request.data.get('stylesheet'), user=self.request.user)
        else:
            serializer.save(user=self.request.user)
```

I also tried 

``` Python
class PageSerializer(serializers.ModelSerializer):
    user = serializers.HiddenField(
        default=serializers.CurrentUserDefault()
    )

    class Meta:
        model = Page
        fields = ('id', 'name', 'stylesheet', 'user', 'created', 'updated')

# with
def perform_create(self, serializer):
    serializer.save(stylesheet=self.request.data.get('stylesheet'))
```

**Comment by Siecje** [comment]
Fixed in [d6c4ce9567e1a9c85ef3d0c28ab65362714c6b98](https://github.com/Siecje/agora-api/commit/d6c4ce9567e1a9c85ef3d0c28ab65362714c6b98)
